From: Hilko Bengen <bengen@debian.org>
Date: Mon, 13 Jan 2020 20:13:30 +0100
Subject: Link .mli files corresponding to autorgenerated .ml files into
 builddir

This avoids build failures in separate builds such as this one:

,----
| ocamlfind ocamlc -package str,unix -I . -a guestfs_config.cmo stringMap.cmo stringSet.cmo std_utils.cmo -o mlstdutils.cma
| ar cr libmlstdutils.a libmlstdutils_a-dummy.o
| ranlib libmlstdutils.a
| File "_none_", line 1:
| Error: Files std_utils.cmo and guestfs_config.cmo
|        make inconsistent assumptions over interface Guestfs_config
| make[4]: *** [Makefile:2580: mlstdutils.cma] Error 2
`----
---
 configure.ac | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure.ac b/configure.ac
index 0db2129..ab110c4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -268,6 +268,9 @@ AC_CONFIG_FILES([Makefile
                  tests/disks/test-qemu-drive-libvirt.xml
                  website/index.html])
 
+AC_CONFIG_LINKS([common/mlstdutils/guestfs_config.mli:common/mlstdutils/guestfs_config.mli
+                 daemon/daemon_config.mli:daemon/daemon_config.mli])
+
 AC_OUTPUT
 
 dnl Produce summary.
